### ISSUE TITLE
After update to 6.5.2 the radar diagram shows no data

### ISSUE THREAD

**Issue by HenrikRolandHansen** [snuids/grafana-radar-panel#19]
After update to 6.5.2 the radar diagram shows no data - I have the latest version of radar panel (34713c1)  Anyone have noticed same problem.  I use Data via influxdb.

**Comment by aerosadegh** [comment]
Yes, I have Grafana v6.6.1 and the radar diagram in my old dashboard shows no data.
I use Data via InfluxDB too!
I have lastest radar panel too!
